dev/com.ibm.ws.security.client_fat/.classpath

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
New projects should be named `io.openliberty...`. In this case `io.openliberty.security.internal.client_fat` or maybe `io.openliberty.security.client.internal_fat`?

Reply:
Technically, this is a project that I moved from WS-CD-Open. Additionally, I name the fats to match the project they are testing. In this case, the project being tested is `com.ibm.ws.security.client`.